java/FoodTruckApp/src/main/java/com/my/FoodTruckApp/appetizer/AppetizerRepository.java

```diff
@@ -1,17 +1,56 @@
 package com.my.FoodTruckApp.appetizer;
-
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.stereotype.Repository;
+import org.springframework.web.server.ResponseStatusException;
 
 import java.util.ArrayList;
-import java.util.Arrays;
 
 @Repository
+@RequiredArgsConstructor
+@Slf4j
 public class AppetizerRepository {
-    Appetizer appetizer1 = new Appetizer(1, "small", "spicy", "dinner", 4);
-    Appetizer appetizer2 = new Appetizer(2, "medium", "sweet", "breakfast", 6);
-    ArrayList<Appetizer> appetizers = new ArrayList<>(Arrays.asList(appetizer1, appetizer2));
 
-    public ArrayList<Appetizer> getAllAppetizers() {
+    private final JdbcTemplate jdbcTemplate;
+    public ArrayList<Appetizer> getListOfAppetizers() {
```

Comment:
super small: can we add a new line here?

Reply:
done!